Apps/W1/ShopifyConnector/app/Shipping/Tables/ShpfyShipmentMethod.Table.al

```diff
@@ -0,0 +1,39 @@
+/// <summary>
+/// Table Shpfy Shipment Method (ID 30131).
+/// </summary>
+table 30131 "Shpfy Shipment Method"
+{
+    Access = Internal;
+    Caption = 'Shopify Shipment Method';
+    DataClassification = CustomerContent;
+
+    fields
+    {
+        field(1; "Shop Code"; Code[20])
+        {
+            Caption = 'Shop Code';
+            DataClassification = SystemMetadata;
+            TableRelation = "Shpfy Shop";
+        }
+        field(2; Name; Text[50])
+        {
+            Caption = 'Name';
+            DataClassification = SystemMetadata;
+            Editable = false;
+        }
+        field(3; "Code"; Code[10])
```

Comment:
please rename field to: "Shipment Method Code"

Reply:
Updated with commit "Object Renaming"